LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2023 Hernán Güemes
+Copyright (c) 2023 Andy Zamorano && Andrea Manuel && Nitecelis Bravo Alcala && Hernán Güemes
```

Comment:
@HFG43 I kindly suggest that you write the names of each team member correctly.
For example, nicknames should not be used in the LICENSE file (`Andy` should be changed to `Andrés`)
Likewise, the name `Nitecelis` is spelled incorrectly, it should be changed to: `Nitcelis`